src/ui/pages/SwapForm/shared/getPopularTokens.ts

```diff
@@ -0,0 +1,21 @@
+import ky from 'ky';
+import { DEFI_SDK_TRANSACTIONS_API_URL } from 'src/env/config';
+import type { Chain } from 'src/modules/networks/Chain';
+import { invariant } from 'src/shared/invariant';
+
+export async function getPopularTokens(chain: Chain) {
+  const params = new URLSearchParams({ chain: chain.toString() });
+  invariant(
+    DEFI_SDK_TRANSACTIONS_API_URL,
+    'DEFI_SDK_TRANSACTIONS_API_URL must be defined in ENV'
+  );
```

Comment:
We can make this check outside the function

Reply:
maybe remove altogether considering we merge this: https://github.com/zeriontech/zerion-wallet-extension/pull/721